### ISSUE TITLE
Broken Install Instructions?

### ISSUE THREAD

**Issue by psema4** [nodester/nodester#409]
Via StackOverflow: http://stackoverflow.com/questions/13615393/lost-in-nodester-installation

On the wiki at  https://github.com/nodester/nodester/wiki/Install-nodester , under Create the Environment:

_Export paths (to make npm work):_

```
> cd ~
> echo -e "root = ~/.node_libraries\nmanroot = ~/local/share/man\nbinroot = ~/bin" > ~/.npmrc
> echo -e "export PATH=3d9c7cfd35d3628e0aa233dec9ce9a44d2231afcquot;\${PATH}:~/bin3d9c7cfd35d3628e0aa233dec9ce9a44d2231afcquot;;" >> ~/.bashrc
> source ~/.bashrc
```

The  exported path line appears... malformed.

According to the original SO question, sourcing bashrc generates the following message:

```
[root@MYSERVER ~]# source ~/.bashrc
-bash: /root/.bashrc: line 13: syntax error near unexpected token ';;'
-bash: /root/.bashrc: line 13: 'export PATH=3d9c7cfd35d3628e0aa233dec9ce9a44d2231afcquot;${PATH}:~/bin3d9c7cfd35d3628e0aa233dec9ce9a44d2231afcquot;;
```

**Comment by alejandro** [comment]
Yeah. The installation instructions are really, really old. I'll be updating the docs this week. Also you can checkout https://github.com/dejl/nodester/tree/light which is a light version of nodester (without chroots). BTW The npm stuff is already solved, you don't need to fix it.

**Comment by slamkajs** [comment]
Following psema4's advice on SO resolved the issue I posted.  I successfully followed all the instructions once I did that.  I am now trying to run the following command:

```
sudo start proxy && sudo start app
```

When I run this, I do get a response.

```
proxy start/running, process 3435
app start/running, process 3441
```

The issue I have though is when I try to access port 80 or port 4001 (not sure if 4001 is where I should be trying as well) I am not getting anything.  I then rerun the start command again and it starts the processes again.

Not sure if this has to do with the issue  at hand but, I figured I would give an update.

**Comment by jfrux** [comment]
Yeah, this is pretty much broken still.  Any ETA?

**Comment by lwille** [comment]
+1 -- the sudoers entries are totally wrong, also it's running /bin/umount as root... is this really necessary? Why not just chroot?

**Comment by legomind** [comment]
+1 wanted to use nodester instead of stagecoach, but without updated install directions, im lost... :-)
